environment/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM almalinux:9 as base
```

Comment:
Please change to `FROM lmalinux:9 AS base`, note the uppercase `AS`, which will remove the Docker warning

```
 1 warning found (use docker --debug to expand):
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)
```